tox.ini

```diff
@@ -34,16 +31,13 @@ usedevelop = false
 
 deps =
   .[test]
-  django111: Django>=1.11,<2.0
-  django20: Django>=2.0,<2.1
-  django21: Django>=2.1,<2.2
   django22: Django>=2.2,<2.3
+  django20: Django>=3.0a1,<3.1
```

Comment:
Should say `django30`

Reply:
Yes, I noticed myself :) I'll push some updates today, and we'll see how it works. It seems that no additional work needs to be done to support Django 3.0. I might be wrong, though.